src/app/components/HtmlClassNameModifier/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import Helmet from 'react-helmet';
+
+const HtmlClassNameModifier = () => {
+  const [htmlClassName, setHtmlClassName] = useState('no-js');
+
+  useEffect(() => {
+    setHtmlClassName('js');
+  }, []);
```

Comment:
This is not a suitable solution. This will always cause a flicker between `no-js` and `js` - meaning the end user will see styling for the former before the latter.